### PR TITLE
DOCSP-43826: v5.2 release

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-43826>
Staging: [compat](https://deploy-preview-175--docs-kotlin.netlify.app/compatibility/), [quickstart](https://deploy-preview-175--docs-kotlin.netlify.app/quick-start/#add-mongodb-as-a-dependency)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/indexes.txt [hunk 1]
```diff
@@ -270,6 +270,11 @@ The following code example shows how to create an Atlas Search index:
 .. literalinclude:: /examples/generated/SearchIndexesTest.snippet.single-search-index-create.kt
    :language: kotlin
 
+To create multiple Search or Vector Search indexes, you must create a
```
Comment (mongoKart) [inline source/fundamentals/indexes.txt:273]: Q: Confirming that these should be capitalized

File: source/includes/language-compatibility-table-kotlin.rst
Comment (mongoKart) [inline source/includes/language-compatibility-table-kotlin.rst:1]: Not part of this PR, but you can also remove MongoDB columns for v3.4 and earlier

**Review by mongoKart — APPROVED** [review]
LGTM w/ 2 notes

File: source/fundamentals/indexes.txt (hunk 1 above)
Comment (rustagir) [reply source/fundamentals/indexes.txt:273]: I think so? seems like product names are capped https://www.mongodb.com/docs/meta/style-guide/style/product-names-version-numbers/
reference: https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-type/
